content/blog/2020/03/2020-03-24-azure-key-vault-cred-provider.adoc

```diff
@@ -0,0 +1,213 @@
+---
+layout: post
+title: "Introducing the Azure Key Vault Credentials Provider for Jenkins"
+tags:
+- jenkins
+- pipeline
+- security
+- azure
+- credentials
+- credential-provider
+- configuration-as-code
+author: timja
+description: "Tightly integrate Jenkins and Azure Key Vault"
```

Comment:
Would be great to add an OpenGraph for social media

Reply:
added, PTAL